docs/docs/version-comparison.md

```diff
@@ -2,59 +2,126 @@
 sidebar_position: 3
 ---
 
-# Version Comparison
-
-You are free to run the Open Source version of Flagsmith however you see fit! There are some differences between the
-Open Source, SaaS hosted and Enterprise versions:
-
-- The Open Source version has **no** API request or Identity limits - you can run as many API instances in a cluster as
-  you wish.
-- The Open Source version has **no** Dashboard User limits.
-- The Open Source version has **no** Environment limits.
-- The Open Source version is limited to a single Project.
-- The SaaS and Enterprise versions have [Change Requests and Flag Scheduling](advanced-use/change-requests.md).
-- The SaaS and Enterprise versions have [Role-Based Access Control](/system-administration/rbac).
-- The SaaS and Enterprise versions have [Audit Logs](/system-administration/audit-logs).
-- The SaaS and Enterprise versions have additional Authentication Providers:
-  - [Okta](/system-administration/authentication/Okta)
-  - [LDAP](/system-administration/authentication/LDAP)
-  - [SAML](/system-administration/authentication/SAML)
-  - [ADFS](/system-administration/authentication/ADFS)
-
-:::tip
-
-You can switch between SaaS and Self Hosted Flagsmith using our
-[Import and Export tools](system-administration/importing-and-exporting/organisations).
+import Card from '@site/src/components/Card';
+import CardHeader from '@site/src/components/Card/CardHeader';
+import CardBody from '@site/src/components/Card/CardBody';
+import CardFooter from '@site/src/components/Card/CardFooter';
 
-:::
-
-## SaaS Benefits
+# Version Comparison
 
-Our SaaS platform has a number of benefits:
+Choose the right Flagsmith version for your needs. Compare our Open Source, SaaS, and Enterprise offerings:
```

Comment:
The original document was all over the place and difficult to follow, but I still consider this new version to be a downgrade overall.

There are 2 dimensions that users need to choose when deciding to use Flagsmith:

1. Self-hosted or SaaS
2. Free or paid plan

These are separate from each other - you can use SaaS or self-host for free (open source version). You can use pay for self-hosting on the Enterprise plan, or pay Flagsmith to host it for you. I feel this document is now a copy of https://www.flagsmith.com/pricing and we don't gain anything by having it.